### PR TITLE
Move all scripts to use hack rather than test-infra

### PR DESCRIPTION
**Pull request by markusthoemmes**
## Proposed Changes

As per title, seems like this repo was missed in the transition to hack scripts. buoy uses that to detect stuff for the release, so that's blocking us currently.

### CODE REVIEW

File: hack/verify-codegen.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ set -o pipefail
 
 export GO111MODULE=on
 
-source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/library.sh
+source $(dirname $0)/../vendor/knative.dev/hack/library.sh
```
Comment (rhuss) [inline hack/verify-codegen.sh:23]:
```suggestion
source $(dirname "$0")/../vendor/knative.dev/hack/library.sh
```

File: test/presubmit-tests.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@
 export GO111MODULE=on
 export DISABLE_MD_LINTING=1
 
-source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/presubmit-tests.sh
+source $(dirname $0)/../vendor/knative.dev/hack/presubmit-tests.sh
```
Comment (rhuss) [inline test/presubmit-tests.sh:27]:
```suggestion
source $(dirname "$0")/../vendor/knative.dev/hack/presubmit-tests.sh
```

File: go.mod [hunk 3]
```diff
@@ -8,12 +8,14 @@ require (
 	github.com/google/go-cmp v0.5.5
 	go.uber.org/zap v1.16.0
 	golang.org/x/net v0.0.0-20210119194325-5f4716e94777
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	k8s.io/api v0.19.7
 	k8s.io/apimachinery v0.19.7
 	k8s.io/client-go v9.0.0+incompatible
+	k8s.io/code-generator v0.19.7
```
Comment (rhuss) [inline go.mod:15]: why is this needed ?

**Review by rhuss — CHANGES_REQUESTED** [review]
minor nits, but looks good in general :)

File: go.mod (hunk 3 above)
Comment (markusthoemmes) [reply go.mod:15]: Because update-codegen.sh depends on it.

**Review by rhuss — APPROVED** [review]
/lgtm
/approve
